base/applications/rapps/unattended.cpp

```diff
@@ -283,18 +283,22 @@ BOOL ParseCmdAndExecute(LPWSTR lpCmdLine, BOOL bIsFirstLaunch, int nCmdShow)
     }
     else if (MatchCmdOption(argv[1], CMD_KEY_FIND))
```

Comment:
```suggestion
    ConInitStdStreams();
    if (MatchCmdOption(argv[1], CMD_KEY_FIND))
```
You'd also need to add a return in the first if() case

Reply:
then you can remove the init from all